File: src/main/java/com/waridley/ttv/logger/backend/local/SlfMessageLogger.java
package com.waridley.ttv.logger.backend.local;

import com.github.twitch4j.chat.events.channel.ChannelMessageEvent;
import com.waridley.ttv.logger.ChatLogger;
import lombok.extern.slf4j.Slf4j;

@Slf4j
public class SlfMessageLogger implements ChatLogger.MessageLogger {
	
	@Override
	public void logMessage(ChannelMessageEvent event) {
		log.info("{} {}... {}", "[" + event.getChannel().getName() + "]", String.format("%-25s", event.getUser().getName()).replace(' ', '.'), event.getMessage());
	}
}

File: src/main/java/com/waridley/ttv/logger/backend/local/SlfMessageLogger.kt
package com.waridley.ttv.logger.backend.local

import com.github.twitch4j.chat.events.channel.ChannelMessageEvent
import com.waridley.ttv.logger.ChatLogger.MessageLogger
import org.slf4j.LoggerFactory

class SlfMessageLogger : MessageLogger {
	override fun logMessage(event: ChannelMessageEvent) {
		log.info("[{}] {}... {}", event.channel.name, String.format("%-25s", event.user.name).replace(' ', '.'), event.message)
	}
	
	companion object {
		private val log = LoggerFactory.getLogger(SlfMessageLogger::class.java)
	}
}